recipe/meta.yaml

```
@@ -12,39 +12,55 @@ source:
   fn: {{ name }}-{{ version }}.{{ bundle }}
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
   sha256: {{ hash }}
+  patches:
+    - patches/0001-loosen-jellyfish-ties.patch
 
 build:
-  noarch: python
+  # no jellyfish>=0.11.2 on s390x
+  skip: True  # [py<36 or s390x]
   entry_points:
     - states = us.cli.states:main
   number: {{ build }}
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - jellyfish ==0.5.6
-    - setuptools
+    - python
+    - jellyfish >=0.11.2
 
 test:
+  source_files:
+    - us/tests
   imports:
     - us
     - us.cli
-
+  requires:
+    - pip
+    - pytest
+    - pytz
   commands:
     - states --help
+    - pip check
+    - pytest -v us/tests
```

```suggestion
    - pytest -v --pyargs us
```
Because the package already contains the tests, we don't need to bundle them. With `ppyargs us`, pytest will automatically find the tests inside the `us` package.